### PR TITLE
Fixes #20848 - Fix error messages indentation

### PR DESCRIPTION
**Pull request by iNecas**
Also fixes #20847 - unify steps description

### CODE REVIEW

**Comment by iNecas** [comment]
Fixed rubocop

File: test/lib/support/definitions/checks/dummy.rb [hunk 1]
```diff
@@ -19,6 +19,17 @@ def run
     end
   end
 
+  class Fail2 < ForemanMaintain::Check
+    metadata do
+      label :dummy_check_fail2
+      description 'check that ends up with fail'
```
Comment (swapab) [inline test/lib/support/definitions/checks/dummy.rb]: Can we follow the `uppercase` in tests as well?

**Comment by iNecas** [comment]
Updated + made the test less dependent on the description in the future (by checking against label instead of description in LogReporter)

**Comment by swapab** [comment]
This PR is good to merge.

**Pull request merged**